src/games/sponge/index.js

```diff
@@ -0,0 +1,353 @@
+            shapeType: Shapes.POLYGON,
+            coordinates2d: ShapeUtils.rectangle(0, 40, 5, 25),
+            fill: COLORS.WHITE
+        });
+
+        const rightPaddle = new GameNode.Shape({
+            shapeType: Shapes.POLYGON,
+            coordinates2d: ShapeUtils.rectangle(95, 40, 5, 25),
+            fill: COLORS.WHITE 
+        });
+
+        const leftScore = new GameNode.Text({
+            textInfo: {
+                text: '0',
+                x: 25,
+                y: 1,
+                color: COLORS.WHITE,
+                size: 4,
+                align: 'center'
+            }
+        });
+
+        const rightScore = new GameNode.Text({
+            textInfo: {
+                text: '0',
+                x: 75,
+                y: 1,
+                color: COLORS.WHITE,
+                size: 4,
+                align: 'center'
+            }
+        });
+
+        this.ball = ball;
+        this.leftPaddle = leftPaddle;
+        this.rightPaddle = rightPaddle;
+        this.leftScore = leftScore;
+        this.rightScore = rightScore;
+
+        this.base.addChildren(this.ball, this.leftPaddle, this.rightPaddle, this.leftScore, this.rightScore); 
+    }
+
+    handleNewPlayer({ playerId, info: playerInfo }) {
+        this.players[playerId] = { playerInfo };
+        this.handlePlayerChange();
+    }
+
+    handlePlayerChange() {
+        const playerCount = Object.keys(this.players).length;
+        if (playerCount > 1) {
+            // update input handlers
+        } else if (playerCount == 1) {
+            // just one
+            const playerId = Object.keys(this.players)[0];
+            this.clickHandlers[playerId] = (x, y) => {
+                if (x < 50) {
+                    // controlling left
+                    this.leftPaddle.node.coordinates2d = ShapeUtils.rectangle(0, Math.min(Math.max(0, y - 12.5), 75), 5, 25); // 25 is paddle height 
+                } else {
+                    // right
+                    this.rightPaddle.node.coordinates2d = ShapeUtils.rectangle(95, Math.min(Math.max(0, y - 12.5), 75), 5, 25); // 25 is paddle height 
+                }
+            };
+            this.startBall();
+        }
+    }
+
+    startBall() {
+        const ballX = 50;
+        const ballY = 50;
+        
+        this.ball.node.coordinates2d = ShapeUtils.rectangle(ballX, ballY, BALL_SIZE, BALL_SIZE);
+
+
+        const xSign = Math.random() < .5 ? -1 : 1;
+        const ySign = Math.random() < .5 ? -1 : 1;
+
+        let randXVel = xSign * Math.floor(Math.random() * 5);
+        let randYVel = ySign * Math.floor(Math.random() * 5);
+
+        if (randYVel == 0) {
+            randYVel = 1;
+        }
+
+        if (randXVel == 0) {
+            randXVel = 1;
+        }
+
+        const ballPath = Physics.getPath(ballX, ballY, randXVel, randYVel, 100 - BALL_SIZE, 100 - BALL_SIZE);
+
+        this.moveBall(ballPath);
+    }
+
+    playSound(dir) {
```

Comment:
do you think this should be a utility thats publicly available. I feel like any game with sound would need something like this